inference-engine/ie_bridges/python/install/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(WHEEL_PACKAGE_NAME "openvino" CACHE STRING "Name of the package")
+set(WHEEL_LICENCE_TYPE "OSI Approved :: Apache Software License" CACHE STRING "License for the package")
+set(WHEEL_AUTOR "Intel Corporation" CACHE STRING "Package author’s name")
+set(WHEEL_AUTOR_EMAIL "openvino_pushbot@intel.com" CACHE STRING "Email address of the package author")
+set(WHEEL_DESC "Inference Engine Python* API" CACHE STRING "Short, summary description of the package")
+set(WHEEL_VERSION "0.0.0" CACHE STRING "Version of this release")
+set(WHEEL_BUILD "000" CACHE STRING "Build number of this release")
+set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in")
+set(SETUP_PY    "${CMAKE_CURRENT_BINARY_DIR}/setup.py")
+set(OVERVIEW_IN "${CMAKE_CURRENT_SOURCE_DIR}/meta/pypi_overview.md")
+set(LICENSE_IN "${CMAKE_CURRENT_SOURCE_DIR}/meta/Apache_license.txt")
+set(REQUIREMENTS_IN "${CMAKE_CURRENT_SOURCE_DIR}/meta/openvino.requirements.txt")
+
+
+if(APPLE)
+    set(WHEEL_PLATFORM macosx_10_15_x86_64)
+elseif(UNIX)
+    set(WHEEL_PLATFORM manylinux2014_x86_64)
+elseif(WIN32)
+    set(WHEEL_PLATFORM win_amd64)
+else()
+    unset(WHEEL_PLATFORM)
+endif()
+
+if (WIN32)
+    file (GLOB LIBRARY_SRC
+         ${LIBRARY_SRC}
+         ${CMAKE_CURRENT_SOURCE_DIR}/os/win/*.cpp)
+    file (GLOB LIBRARY_HEADERS
+         ${LIBRARY_HEADERS}
+         ${CMAKE_CURRENT_SOURCE_DIR}/os/win/*.hpp)
+endif()
```

Comment:
I don't see such platforms specific files in source tree..

Reply:
Sorry, this is old code(

Reply:
fixed